src/pudl_archiver/archivers/epacems.py

```diff
@@ -61,48 +61,55 @@ class EpaCemsArchiver(AbstractDatasetArchiver):
     base_url = "https://api.epa.gov/easey/bulk-files/"
     parameters = {"api_key": os.environ["EPACEMS_API_KEY"]}  # Set to API key
 
+    def __filter_for_complete_metadata(
```

Comment:
Thinking aloud here - the only issue here could be if the underlying format of the metadata changes (e.g., year becomes string type), and we'd silently lose all the files. But I guess this would be caught in our other validation tests (e.g. check missing files).